src/js/models/v2/awardsV2/BaseAwardAmounts.js

```diff
@@ -60,6 +60,16 @@ const BaseAwardAmounts = {
             return `${MoneyFormatter.formatMoneyWithPrecision((this._obligation - this._combinedCurrentAwardAmounts) / units.unit, 1)} ${units.unitLabel}`;
         }
         return MoneyFormatter.formatMoney(this._obligation - this._combinedCurrentAwardAmounts);
+    },
+    get extremeOverspending() {
+        return MoneyFormatter.formatMoneyWithPrecision(this._obligation - this._combinedPotentialAwardAmounts, 2);
+    },
+    get extremeOverspendingFormatted() {
```

Comment:
Where are we using these?

Reply:
`extremeOverspendingFormatted` is used [here](https://github.com/fedspendingtransparency/usaspending-website/blob/feature-obligated-exceeds-potential/src/js/components/awardv2/visualizations/amounts/charts/ExceedsPotentialChart.jsx#L63).